src/metadatas.rs

```diff
@@ -46,6 +47,7 @@ pub fn extract_metadata(gtfs: &gtfs_structures::RawGtfs) -> Metadata {
         .minmax()
         .into_option();
     let format = |d: chrono::NaiveDate| d.format("%Y-%m-%d").to_string();
+    let validator_version = env!("CARGO_PKG_VERSION");
```

Comment:
this is never updated is it (or it has been only twice :p)?

There are also trick to get the `git describe` (tag+sha1) to always get a precise version if you're interested.

Reply:
I intended to bump the version from now on, every time a change is made. What is the trick you are talking about?